### DISCUSSION TITLE
Anyone building standalone WatchOS apps successfully?

### DISCUSSION THREAD

**Discussion by bradmartin** [fastlane/fastlane#20117] [Q&A]
For context: https://github.com/fastlane/fastlane/issues/20066

Seems like something is missing on the setup we have on various machines to get fastlane to build correctly. Same Macs build, archive fine inside of XCode with no issue. I've tried specifying the various devices in the script but the error will remain and can't seem to get past that part of the process.

Any insight would be appreciated.

**Comment by fguchelaar** [comment]
HI, did you find a solution for this in the mean time? I tried today, but couldn't get it to work:

>  xcodebuild: error: Unable to find a destination matching the provided destination specifier

**Reply by fguchelaar** [reply]
Got it. You can specify the destination:

```
build_app(
    destination: "generic/platform=watchOS"
)
```
